pkg/virt-launcher/virtwrap/libvirtxml/convert.go

```diff
@@ -0,0 +1,336 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 The KubeVirt Authors
+ *
+ */
+
+package libvirtxml
+
+import (
+	"strconv"
+
+	"libvirt.org/go/libvirtxml"
+
+	api "kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
+)
+
+// This package provides the methods to convert the types defined in
+// kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api to the types defined in
+// libvirt.org/go/libvirtxml package.
+//
+// The libvirtxml package provides 100% coverage to Libvirt XML schemas and we
+// can rely on the Marshal/Unmarsh functions from this package.
+//
+// TODO: remove all the convert functions once we switch using the types of libvirtxml.
+// See: https://github.com/kubevirt/kubevirt/issues/10844
+
+func ConvertKubeVirtCPUFeatureToDomainCPUFeature(features []api.CPUFeature) []libvirtxml.DomainCPUFeature {
```

Comment:
Looks like the only externally used functions are:
- ConvertKubeVirtCPUToDomainCPU
- ConvertKubeVirtVCPUToDomainVCPU
- ConvertKubeVirtCPUTuneToDomainCPUTune
- ConvertKubeVirtNUMATuneToDomainNUMATune
- ConvertKubeVirtFeaturesToDomainFeatureList

How about making all the others unexported?